### DISCUSSION TITLE
IdeaStatica Checkbot SAF file import v23.1

### DISCUSSION THREAD

**Discussion by DallasLee-rbg** [idea-statica/ideastatica-public#358] [Help Q&A] [answered]
Hi,
I've imported a SAF file into checkbot. Inside checkbot all the load cases are showing and there's values for the various actions. However, when I open a particular connection, not all load cases are showing in the connection file. For example, in one model there are 29 load cases showing in checkbot which is correct, but there's only 14 load cases in the showing in the connection file. This has only started happening since using v23.1. Was there something changed in v23.1 that would be causing this?

**Comment by sonkaste1** [comment, marked as answer]
Hi @DallasLee-rbg ,

can you check the "evaluate critical effects" toggle in "Loads" dialogue, please?

1) "Loads" dialogue can be opened from the ribbon
2) Uncheck "Evaluate critical effects" toggle for having all loads that are in the "Result class" imported to Connection
3) Save
4) Update Load effects for Connection and Open Connection

![image](https://github.com/idea-statica/ideastatica-public/assets/87471266/7865a987-3f24-4879-928b-f2675defd6ae)
